.travis.yml

```diff
@@ -18,7 +18,7 @@ before_install:
   - composer selfupdate
 
 install:
-  - travis_retry composer install --no-interaction --prefer-source
+  - composer install --no-interaction --prefer-dist
```

Comment:
You definely don't want that. Using that is just selfishly using api calls for that mashine.